recipes/unishox2-py3/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "unishox2-py3" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/unishox2-py3-{{ version }}.tar.gz
+  sha256: cd2595de196ffd8b93bea7a76f5796d8293f32d1f7c9d3660af0b764ab917963
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win or arm64]
+  number: 1
+
+requirements:
+  build:
+    - python
+    - pip
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - unishox2
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/tweedge/unishox2-py3
+  summary: A package for Unicode-friendly string compression using Unishox2
+  dev_url: https://github.com/tweedge/unishox2-py3
+  license: Apache-2.0
+  license_file:
+    - LICENSE
```

Comment:
This library is revendoring unishox2 which is not allowed. Typically, we would ask you to devendor unishox2 into its own package and dynamically link, but since unishox2 is so small we will treat it as a header-only library.

In this case, as explicitly mentioned in the checklist, you must include the license of the statically-linked/revendored library. The license of unishox2 and unishox2-py3 are different because the license *holders* are not the same, so please include both.